plugin/testdata/http-log.lua

```diff
@@ -0,0 +1,55 @@
+local typedefs = require "kong.db.schema.typedefs"
+local url = require "patched.url"
```

Comment:
Stole this from the patch as well; wonder if there is something we can do which circumvents all the known modules used by Kong Gateway which use C libraries? (just thinking out loud here for the future).

Reply:
Maybe, when I did all of this, my goal was to keep the patches to a minimum.